test/model/workloads/WT-15311.workload

```diff
@@ -0,0 +1,25 @@
+config("database", "disaggregated=true")
```

Comment:
I thought this was currently failing with disagg true?

Reply:
currently it fails because there is no database created